services/motion/builtin/builtin_test.go

```diff
@@ -1011,6 +1020,9 @@ func TestStoppableMoveFunctions(t *testing.T) {
 			slamName := "test-slam"
 
 			// Create an injected SLAM
+			// NOTE: octagon.pcd's bounding circle has a radius less than 1.3 meters. This means that when using this .pcd
+			// all goals will be less than 2.6 meters away. For this reason we must specify our own PlanDeviationMM value
+			// within the MotionConfiguration.
```

Comment:
I think we can get rid of all these comments.  It is reasonably obvious now that the changes are necessary given the location of the goal